borsh-schema-derive-internal/src/snapshots/borsh_schema_derive_internal__enum_schema__tests__complex_enum_generics_borsh_skip_named_field.snap

```diff
@@ -2,12 +2,15 @@
 source: borsh-schema-derive-internal/src/enum_schema.rs
 expression: pretty_print_syn_str(&actual).unwrap()
 ---
-impl<C, W> borsh::BorshSchema for A<C, W>
+impl<W, U, C> borsh::BorshSchema for A<W, U, C>
 where
+    U: borsh::BorshSchema,
     C: borsh::BorshSchema,
 {
     fn declaration() -> borsh::schema::Declaration {
-        let params = borsh::__private::maybestd::vec![< C > ::declaration()];
+        let params = borsh::__private::maybestd::vec![
+            < U > ::declaration(), < C > ::declaration()
```

Comment:
the order has been changed to follow not alphabetic order of types transcription, but 
the original order of generics declaration in this [commit](https://github.com/near/borsh-rs/pull/180/commits/1f1a8f4be70fa7bc8bc02748be32b331c1eb3903)